gatorgrade/output/output_functions.py

```diff
@@ -1,7 +1,57 @@
-"""Add docstring."""
+"""
+This class is used for storing the main functions requested from the Github.

```

Comment:
This docstring should match what is located in the command-receive-function branch so that we don't have a merge conflict when combining the two.

The docstring in that branch is:

```
"""This module is used for storing the main functions requested.

The requested functions are located at the Github Issue Tracker
for the output team. For instance, functions dealing with percentage
output, description output, and colorization of text.
"""
```

Reply:
I decided it was best to use the docstring @gvanzin-allegheny provided given that it will be more consistent, and I feel it is more descriptive than what I had.